lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -52,7 +54,14 @@ $members_before = Get-AdGroupMember -Identity $name @extra_args
 $pure_members = [System.Collections.Generic.List`1[String]]@()
 
 foreach ($member in $members) {
-    $group_member = Get-ADObject -Filter "SamAccountName -eq '$member' -and $ad_object_class_filter" -Properties objectSid, sAMAccountName @extra_args
+    $extra_member_args = $extra_args.Clone()
+    if ($member -match "\\"){
+        $extra_member_args.Server = $member.Split("\")[0]
+        $member = $member.Split("\")[1]
+    } else {
+        $extra_member_args = $extra_args
```

Comment:
Not needed as you have already cloned `$extra_args` to `$extra_member_args`.

Reply:
i removed the part and tested it successfully